bitnami/oauth2-proxy/values.yaml

```diff
@@ -358,6 +358,7 @@ configuration:
   ## @param configuration.google.existingSecret Existing secret containing Google Service Account
   ##
   google:
+    useApplicationDefaultCredentials: false
```

Comment:
You need to add the metadata for this new parameter, see the above metadata corresponding to the already existing parameters.

Reply:
i belive is ok now